corehq/apps/app_manager/xform_builder.py

```diff
@@ -168,15 +168,15 @@ def get_text_id(name, groups=None, choice_name=None, is_hint=False):
         Builds a text node "id" parameter
 
         >>> XFormBuilder.get_text_id('foo')
-        'foo-label'
+        u'foo-label'
```

Comment:
:grey_question: 
Is there no way to use `six` in doctests?

Reply:
I haven't tried, but that would probably work.  I'm not sure that it would be much better to use ```six.text_type('foo-label')``` vs ```u'foo-label'``` though.  Either way it would be a good idea to cleanup after upgrading to py3.